e2e/cypress/support/api_commands.js

```diff
@@ -280,6 +280,24 @@ Cypress.Commands.add('apiSaveUserPreference', (preferences = [], userId = 'me')
     });
 });
 
+/**
+ * Saves clock display mode 24-hour preference of a user directly via API
+ * This API assume that the user is logged in and has cookie to access
+ * @param {Boolean} is24Hour - Either true (default) or false
+ */
+Cypress.Commands.add('apiSaveClockDisplayMode24HourPreference', (is24Hour = true) => {
```

Comment:
Once move as commented above, please add type as well.

Reply:
added to type definition. moved all preferences to preference.js but only added type definition to `apiSaveUserPreference` and `apiSaveClockDisplayModeTo24HourPreference` for now. The rest of the type definition can be added in a different PR